hw_4/hw_4.3.js

```diff
@@ -0,0 +1,22 @@
+/*
+Task 3.
+Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+*/
+
+let age;
+age = prompt('Enter your age.');
+
+const age_2 = 18;
+const age_3 = 60;
+
+if (!Number.isInteger(+age) || age < 0 || Boolean(+age) === false) {
```

Comment:
Same situation as for the above comment. Please update checks

Reply:
Not at all.
I covered cases for age = ‘’, age = ‘  ‘, age = null, etc.
It works quite well. 
Could you launch it via Live Server?

Reply:
As per my understanding you need the next flow:
1. Check isNaN(age). If true - provide error
2. Convert age to number (if it is number as well - nothing will happen)
3. Do regular checks for different ages

Reply:
done